.circleci/config.yml

```diff
@@ -97,7 +97,7 @@ jobs:
             . env/bin/activate
             python -m build \
               -Ccompile-args="-v" \
-              -Csetup-args="-Dbuildtype=debug" \
+              -Csetup-args="-Db_ndebug=true" \
```

Comment:
the `bdebug` build type builds the C++ test, which doesn't work with creating a wheel. So we instead turn on debug symbols directly.